projects/ngx-datatable/src/lib/ngx-datatable.module.ts

```diff
@@ -77,6 +77,16 @@ export interface INgxDatatableConfig {
     totalMessage: string;
     /** Footer selected message */
     selectedMessage: string;
+    /** Pager screen reader message for the first page button */
+    ariaFirstPageMessage: string;
+    /** Pager screen reader message for the n-th page button */
+    ariaPageMessage: string;
```

Comment:
I think we need a more descriptive message here.

```suggestion
    /** 
      * Pager screen reader message for the n-th page button. 
      * It will be rendered as: `{{ariaPageNMessage}} {{n}}`.
      */
    ariaPageNMessage: string;
```